openmdao/core/group.py

```diff
@@ -3051,7 +3051,35 @@ def _setup_auto_ivcs(self, mode):
             if not remote and 'value' in gmeta:
                 val = gmeta['value']
             relsrc = src.rsplit('.', 1)[-1]
-            auto_ivc.add_output(relsrc, val=val, units=units)
+
+            # Add the output quickly.
+            # We don't need to check for errors because we get the value straight from a
+            # source, and ivc metadata is minimal.
```

Comment:
I think it might be cleaner to just override the `add_output` method on `_AutoIndepVarComp`

Reply:
I moved it into _AutoIndepVarComp, which negates the other finding. (As long as @naylor-b is okay with it.)

Reply:
It's ok with me.